templates/role.tmpl

```diff
@@ -27,36 +27,128 @@
-                                <li style='list-style-type: none;'><span class='tab'></span>
-                                    {{if $userAdmin}}
-                                        <a href="/internal/user/{{.UserID}}">{{formatUserNameUserStruct .}}</a>
-                                    {{else}}
-                                        {{formatUserNameUserStruct .}}
-                                    {{end}}
-                                </li>
-                            {{end}}
-                        </ol>
+                    {{if gt (len .Users) 0}}<br><br>
+                    Inherited by:
+                    <ol>
+                        {{range .Users}}
+                            <li style='list-style-type: none;'><span class='tab'></span>
+                                {{if $userAdmin}}
+                                    <a href="/internal/user/{{.UserID}}">{{formatUserNameUserStruct .}}</a>&emsp;<a
+                                            class="button is-danger is-outlined"
+                                            onclick="removeUser{{.UserID}}FromRoleModal()">Remove user</a>
+                                {{else}}
+                                    {{formatUserNameUserStruct .}}&emsp;<a
+                                            class="button is-danger is-outlined"
+                                            onclick="removeUser{{.UserID}}FromRoleModal()">Remove user</a>
+                                {{end}}
+                            </li>
+                            <div id="removeUser{{.UserID}}FromRoleModal" class="modal">
+                                <div class="modal-background"></div>
+                                <div class="modal-content">
+                                    <div class="box">
+                                        <article class="media">
+                                            <div class="media-content">
+                                                <div class="content"> <p class="title">Are you sure you want to remove "{{formatUserNameUserStruct .}}" from this
+                                                        role?</p>
+                                                    <form action="/internal/role/{{$.Role.RoleID}}/user/remove/{{.UserID}}" method="post">
+                                                        <button class="button is-danger">Remove user</button>
+                                                    </form>
+                                                </div>
+                                            </div>
+                                        </article>
+                                    </div>
+                                </div>
+                                <button class="modal-close is-large" aria-label="close"></button>
+                            </div>
+                            <script>
+                                function removeUser{{.UserID}}FromRoleModal() {
```

Comment:
This will create a function for every user ID (that has roles) - are you sure we want to do that? Could make a `function removeUserFromRoleModal(userID) {}`.

Reply:
Will fix this